src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -215,61 +217,6 @@ export function CollapsibleNav({
       outsideClickCloses={false}
     >
       <EuiFlexItem className="eui-yScroll">
-        {/* Recently viewed */}
-        <EuiCollapsibleNavGroup
-          key="recentlyViewed"
-          background="light"
-          title={i18n.translate('core.ui.recentlyViewed', { defaultMessage: 'Recently viewed' })}
-          isCollapsible={true}
-          initialIsOpen={getIsCategoryOpen('recentlyViewed', storage)}
-          onToggle={(isCategoryOpen) =>
-            setIsCategoryOpen('recentlyViewed', isCategoryOpen, storage)
-          }
-          data-test-subj="collapsibleNavGroup-recentlyViewed"
-        >
-          {recentlyAccessed.length > 0 ? (
-            <EuiListGroup
-              aria-label={i18n.translate('core.ui.recentlyViewedAriaLabel', {
-                defaultMessage: 'Recently viewed links',
-              })}
-              listItems={recentlyAccessed.map((link) => {
-                // TODO #64541
-                // Can remove icon from recent links completely
-                const { iconType, onClick, ...hydratedLink } = createRecentNavLink(
-                  link,
-                  navLinks,
-                  basePath,
-                  navigateToUrl
-                );
-
-                return {
-                  ...hydratedLink,
-                  'data-test-subj': 'collapsibleNavAppLink--recent',
-                  onClick: (event) => {
-                    if (!isModifiedOrPrevented(event)) {
-                      closeNav();
-                      onClick(event);
-                    }
-                  },
-                };
-              })}
-              maxWidth="none"
-              color="subdued"
-              gutterSize="none"
-              size="s"
-              className="osdCollapsibleNav__recentsListGroup"
-            />
-          ) : (
-            <EuiText size="s" color="subdued" style={{ padding: '0 8px 8px' }}>
```

Comment:
Will we still show `No recently viewed items` after this refactor?

Reply:
No. This edge case recently viewed and favorites need to be confirmed with UX.

Reply:
Just get to know that `No recently viewed items` need to show. I will add it later.